sdk/src/main/java/ch/opentransportdata/ojp/data/dto/response/tir/situations/TextualContentDto.kt

```diff
@@ -25,7 +25,9 @@ data class TextualContentDto(
     @Element(name = "siri:DurationContent")
     val durationContent: DurationContentDto?,
     @Element(name = "siri:RemarkContent")
-    val remarkContent: List<RemarkContent>?
+    val remarkContent: List<RemarkContent>?,
+    @Element(name = "siri:InfoLink")
+    val infoLink: InfoLink?
```

Comment:
should be a list, according to: https://vdvde.github.io/OJP/develop/documentation-tables/siri.html#type_siri__TextualContentStructure